app/client/packages/design-system/ads/src/Box/Box.tsx

```
@@ -1,29 +1,35 @@
+    const transformedRest = Object.entries(rest).reduce((acc, [key, value]) => {
+      const newValue = PropsToBeCssPrefixPrepended.includes(key)
+        ? `var(${CSS_VARIABLE_PREFIX}${value})`
+        : value;
 
-    return {
-      ...acc,
-      [key]: newValue,
-    };
-  }, {} as BoxProps);
+      return {
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)